libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -763,6 +763,23 @@ void AC_PosControl::init_xy_controller()
     init_ekf_xy_reset();
 }
 
+/// standby_xyz_reset - resets I terms and removes position error
+///     This function will let Loiter and Alt Hold continue to operate
+///     in the event that the flight controller is in control of the
+///     aircraft when in standby.
+void AC_PosControl::standby_xyz_reset()
+{
+    _pid_accel_z.set_integrator(0.0f);
+
+    _pos_target = _inav.get_position();
+
+    // initialise I terms from lean angles
```

Comment:
is this an out-of-date comment or is it missing the call to "lean_angles_to_accel"?

Reply:
I have added comments for each line and corrected that one.